integration_tests/test_benchmark.py

```diff
@@ -1,16 +1,20 @@
 # Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 # SPDX-License-Identifier: Apache-2.0
-"""Compare benchmark results before and after a pull request."""
+"""
+Compare benchmark results before and after a pull request.
+
+This test works properly on the local machine only when the options
+`--base-branch` and `--remote` are set. Otherwise the default values
+are "master" for the name of the base branch and "origin" for the
```

Comment:
"master" will have to be updated if we switch to the `main` default.

Reply:
I think we should stick to "master" until the update.